presto-docs/src/main/sphinx/release/release-0.222.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.222
+=============
+
+General Changes
+---------------
+* Fix incorrect results when dividing certain decimal numbers.
+* Fix query failure for certain join queries with "Equi criteria are empty,
```

Comment:
I wonder if we could more fully describe this and not have to include the error message here

Reply:
There isn't a good way to describe the kinds of queries that would enounter this error, since it mostly depends on internal implementation details.  But what about: 
```
Fix planning failure for certain join queries caused by invalid distribution type.
```

Reply:
That sounds good to me.